src/components/mainEvent/index.jsx

```diff
@@ -7,6 +7,7 @@ import LocationIcon from '@material-ui/icons/LocationOn';
 import CalendarIcon from '@material-ui/icons/DateRange';
 import ContainerLayout from '../containerLayout';
 import imageHack from '../../images/background.jpg';
+// import AppBar from '@material-ui/core/AppBar';
```

Comment:
why are you adding commented out code? Please see https://github.com/OpenUpSA/best-practices/blob/master/development.md#source-code